mteb/evaluation/MTEB.py

```diff
@@ -379,38 +435,63 @@ def run(
         original_tasks = (
             self.tasks.copy()
         )  # save them in case we re-use the object (e.g. for reranking)
+
+        # To evaluate missing splits, we keep track of the task name and the corresponding splits.
+        # TODO: track languages too.
```

Comment:
by language here you mean the subset on HF?

Reply:
I think this is important, but I can work on after merge this PR

Reply:
Great!